frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -1,208 +1,283 @@
+    const today = curr.toISOString().substr(0, 10);
+    const DefaultVals = {
+        name: "", startingBalance: "10000", cowPrice: "100",
+        milkPrice: "1", degradationRate: null, carryingCapacity: null, startingDate: today
+    };
+
+    const belowStrategy = initialCommons?.belowCapacityStrategy || healthUpdateStrategies?.defaultBelowCapacity;
+    const aboveStrategy = initialCommons?.aboveCapacityStrategy || healthUpdateStrategies?.defaultAboveCapacity;
+
+
+    return (
+        <Form onSubmit={handleSubmit(submitAction)}>
+            {initialCommons && (
+                <Form.Group className="mb-3">
+                    <Form.Label htmlFor="id">Id</Form.Label>
+                    <Form.Control
+                        data-testid={`${testid}-id`}
+                        id="id"
+                        type="text"
+                        {...register("id")}
+                        value={initialCommons.id}
+                        disabled
+                    />
+                </Form.Group>
+            )}
+
+            <div className="border-bottom mb-3"></div>
+
+            <Row className="flex justify-content-start" style={{width: '80%'}} data-testid={`${testid}-r0`}>
+                <Col className="" md={6}>
+                    <Form.Group className="mb-3">
+                        <Form.Label htmlFor="name">Commons Name</Form.Label>
+                        <OverlayTrigger
+                            placement="top"
+                            overlay={<Tooltip>Enter Name for a new common</Tooltip>}
+                            delay='5'
+                        >
+                            <Form.Control
+                                data-testid={`${testid}-name`}
+                                id="name"
+                                type="text"
+                                defaultValue={DefaultVals.name}
+                                isInvalid={!!errors.name}
+                                {...register("name", {required: "Commons name is required"})}
+                            />
+                        </OverlayTrigger>
+                        <Form.Control.Feedback type="invalid">
+                            {errors.name?.message}
+                        </Form.Control.Feedback>
+                    </Form.Group>
+                </Col>
+                <Col md={6}>
+                    <Form.Group className="mb-3">
+                        <Form.Label htmlFor="startingBalance">Starting Balance</Form.Label>
+                        <OverlayTrigger
+                            placement="top"
+                            overlay={<Tooltip>Unit: $</Tooltip>}
+                            delay='100'
+                        >
+                            <Form.Control
+                                id="startingBalance"
+                                data-testid={`${testid}-startingBalance`}
+                                type="number"
+                                step="0.001"
```

Comment:
Starting Balance step should still be 0.01, not 0.001.

It's degradation rate where we want a step value  of 0.001 (in fact, Prof. de Vries now wants `0.0001` instead.  But only for degradation rate.)